spec/factories/case_contact_topics.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :case_contact_topic do
+    title { Faker::Lorem.sentence }
+    details { Faker::Lorem.paragraph }
```

Comment:
You need to make sure you are generating more than 256 characters here.
https://faker.readthedocs.io/en/master/providers/faker.providers.lorem.html